### DISCUSSION TITLE
New airports after MSFS2020 WU8

### DISCUSSION THREAD

**Discussion by AD8448** [albar965/littlenavmap#874] [General]
Hi,
I can see and choose new airports after WU8 in FS2020 world map, but not in LNM, even after updating the library connected to MSFS.
Any solution ?

**Comment by albar965** [comment]
Do you have a few examples?

**Comment by AD8448** [comment]
Hi,

Many thanks for you answer. I appreciate very much LNM.
Some examples :

LPJO
LESH
LEND
LETZ


De : Alexander Barthel ***@***.***>
Envoyé : jeudi, 31 mars 2022 11:52
À : albar965/littlenavmap ***@***.***>
Cc : AD8448 ***@***.***>; Author ***@***.***>
Objet : Re: [albar965/littlenavmap] New airports after MSFS2020 WU8 (Discussion #874)


Do you have a few examples?

**Comment by albar965** [comment]
Thanks. I'll check. Issue is #875.
But it's unlikely that I will create a new LNM release. I'll fix this for one of the next alpha versions.

**Comment by AD8448** [comment]
Thanks to you

De : Alexander Barthel ***@***.***>
Envoyé : jeudi, 31 mars 2022 12:59
À : albar965/littlenavmap ***@***.***>
Cc : AD8448 ***@***.***>; Author ***@***.***>
Objet : Re: [albar965/littlenavmap] New airports after MSFS2020 WU8 (Discussion #874)


Thanks. I'll check. Issue is #875<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Falbar965%2Flittlenavmap%2Fissues%2F875&data=04%7C01%7C%7C6680f50537a447d6290108da1305766a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637843211433304187%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=PLQXxGI2G%2F6P0oGIfC1D4ULZFLjeLeE%2B3%2FWdAkvO8Bg%3D&reserved=0>.
But it's unlikely that I will create a new LNM release. I'll fix this for one of the next alpha versions.

-

**Comment by AD8448** [comment]
Hi,
I found the cause of the problem : after deactivating Navigraph from the LNM library, it see the airports. Before that, I could choose LEVD on the LNM map as departure of the bushtrip, even if the airport was not in the database. For this reason the first leg did not appear in BTI, adding to confusion ! It seems that this bug did not appear before WU8. Thanks for your help. I will inform A. Barthel. Thanks for your help.

De : Alexander Barthel ***@***.***>
Envoyé : jeudi, 31 mars 2022 12:59
À : albar965/littlenavmap ***@***.***>
Cc : AD8448 ***@***.***>; Author ***@***.***>
Objet : Re: [albar965/littlenavmap] New airports after MSFS2020 WU8 (Discussion #874)


Thanks. I'll check. Issue is #875<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Falbar965%2Flittlenavmap%2Fissues%2F875&data=04%7C01%7C%7C6680f50537a447d6290108da1305766a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637843211433304187%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=PLQXxGI2G%2F6P0oGIfC1D4ULZFLjeLeE%2B3%2FWdAkvO8Bg%3D&reserved=0>.
But it's unlikely that I will create a new LNM release. I'll fix this for one of the next alpha versions.

-

**Comment by albar965** [comment]
I see. Thanks for the feedback.
This might be related to the change albar965/atools@41306c366f1ec3ec4af230db9b8722fba57d6232
LNM detected some Navigraph packages by name and these names have changed potentially causing this issue.
It should work without excluding Navigraph for the 2.7 alpha.

**Comment by AD8448** [comment]
I sent you a copy of my message to BuffyGC - BTI - who could of course not understand what was happening. Many thanks for your LNM

De : Alexander Barthel ***@***.***>
Envoyé : vendredi, 1 avril 2022 12:26
À : albar965/littlenavmap ***@***.***>
Cc : AD8448 ***@***.***>; Author ***@***.***>
Objet : Re: [albar965/littlenavmap] New airports after MSFS2020 WU8 (Discussion #874)


I see. Thanks for the feedback.
This might be related to the change ***@***.***<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Falbar965%2Fatools%2Fcommit%2F41306c366f1ec3ec4af230db9b8722fba57d6232&data=04%7C01%7C%7Cb8d7023977b44089845508da13ca131d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637844055874064156%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=lYfoYqw88Iq3bPAWuL%2Brzq5DgkNtqMzo%2BtrqxbPccEQ%3D&reserved=0>
LNM detected some Navigraph packages by name and these names have changed potentially causing this issue.
It should work without excluding Navigraph for the 2.7 alpha.

-

**Comment by albar965** [comment]
This seems to be fixed in the alpha 2.7.8 https://www.avsim.com/forums/topic/614851-little-navmap-alphadevelopment-278-released-update-one/

**Comment by AD8448** [comment]
Many thanks

De : Alexander Barthel ***@***.***>
Envoyé : samedi, 2 avril 2022 15:43
À : albar965/littlenavmap ***@***.***>
Cc : AD8448 ***@***.***>; Author ***@***.***>
Objet : Re: [albar965/littlenavmap] New airports after MSFS2020 WU8 (Discussion #874)


This seems to be fixed in the alpha 2.7.8 https://www.avsim.com/forums/topic/614851-little-navmap-alphadevelopment-278-released-update-one/<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.avsim.com%2Fforums%2Ftopic%2F614851-little-navmap-alphadevelopment-278-released-update-one%2F&data=04%7C01%7C%7C4dd77bf0fe4f4743e85308da14aeb522%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637845037841516699%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qSEJdTn%2FGIa65weZxNN0AzFpxIwJ6hli6lQT1SuXcr0%3D&reserved=0>

-

**Reply by albar965** [reply]
> Many thanks

You're welcome. You can try it in the current or next alpha. Just watch the pinned thread in the Avsim forum.

**Comment by AD8448** [comment]
Hi, I will try. It is not clear in the manual which directory has to be deleted.
In case of interest I can help to translate to French. I would need to know to do, and I cannot begin before next week

De : Alexander Barthel ***@***.***>
Envoyé : dimanche, 3 avril 2022 20:55
À : albar965/littlenavmap ***@***.***>
Cc : AD8448 ***@***.***>; Author ***@***.***>
Objet : Re: [albar965/littlenavmap] New airports after MSFS2020 WU8 (Discussion #874)


Many thanks

You're welcome. You can try it in the current or next alpha. Just watch the pinned thread in the Avsim forum.

-

**Reply by albar965** [reply]
This is fixed in the stable 2.6.19 and in the 2.8.0.beta. No need to delete folders if using the stable version.
